### PR TITLE
Check network is active after creation

### PR DESCRIPTION
**Pull request by nvgoldin**
This will busy-wait until the network is created
for at most 10 seconds. At the minimum, it will
exit after the first check the network is active,
without sleeping.

Signed-off-by: Nadav Goldin <ngoldin@redhat.com>

### CODE REVIEW

**Comment by nvgoldin** [comment]
ci test please

File: lago/virt.py [hunk 1]
```diff
@@ -348,14 +348,50 @@ def resolve(self, name):
...
+                if net is None:
+                    raise RuntimeError(
+                        'failed to create network, XML: %s' %
+                        (self._libvirt_xml())
+                    )
+                for _ in range(attempts):
+                    if net.isActive():
+                        return
+                    LOGGER.debug(
+                        'waiting for network %s to become active', net.name()
```
Comment (mykaul) [inline lago/virt.py]: Is it self.name() or net.name() ? same below.

Comment (nvgoldin) [inline lago/virt.py]: took it directly from libvirt instead(we already know its not ``None`` here..)

**Comment by nvgoldin** [comment]
ci merge please

**Pull request merged**